/remap_manager:
  ros__parameters:
    voxel_size: 0.1
    vertex_centered: false
    fixed_frame: "map"
